docs/pages/components/select.md

```diff
@@ -23,24 +23,24 @@ For lists that require more than 12 options, the <a href="/patterns/combobox-inp
 <div class="fd-popover">
   <div class="fd-popover__control">
      <div class="fd-select">
-         <div class="fd-select__control" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
+         <div class="fd-select__control" role="button" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
              Select
-             <button class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></button>
+             <span class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></span>
```

Comment:
We shouldn't do this either - these docs are supposed to be the source of truth for html markup.

Reply:
Also markup they would need to follow, because the `button` adds an extra tab stop.